TrustModel.md

```diff
@@ -30,6 +30,9 @@ Confidential Computing. We explore trust using different orthogonal ways of cons
 native approaches when they use an underlying TEE technology and 
 identifying where there may be considerations to preserve the value of using a TEE.
 
+### Trust Model Considerations
```

Comment:
Speaking of diagrams, it might be cool to take all these cloud native personas and make a graph where the parties are connected by edged that show their technical and trust relationships.

Reply:
Yea I like the idea of a graph linking the personas, I'll have a think

Reply:
I like as well the idea of having illustrations about how the personas interact. This further could help to understand the operational aspects that comes along with CC.